admin_guide/quota.adoc

```diff
@@ -154,13 +170,13 @@ error.
 |Match pods where `spec.activeDeadlineSeconds` is `nil`.
 
 |*BestEffort*
-|Match pods that have best effort quality of service for either `*cpu*` or
-`*memory*`. See the xref:../admin_guide/overcommit.adoc#qos-classes[Quality of
+|Match pods that have best effort quality of service for either `*cpu*`,
+`*memory*`, or `*ephemeral-storage*`. See the xref:../admin_guide/overcommit.adoc#qos-classes[Quality of
```

Comment:
Ephemeral storage is not tied to quality of service

Reply:
OK